### PR TITLE
#548 redundant email on message post

### PR DESCRIPTION
**Pull request by mbarbieri**

### CODE REVIEW

**Comment by karato** [comment]
@mbarbieri Thanks for the pull request, let me find a reviewer..

**Comment by karato** [comment]
@caarlos0 review it

**Comment by caarlos0** [comment]
@mbarbieri you need to describe your changes or QA will complain..

File: netbout-web/src/test/java/com/netbout/email/EmAliasTest.java [hunk 1]
```diff
@@ -74,4 +75,34 @@ public void sendsEmail() throws Exception {
...
+            msg.getAllRecipients().length,
+            Matchers.is(1)
+        );
+        final InternetAddress recipientAddress =
+            (InternetAddress) msg.getAllRecipients()[0];
+        MatcherAssert.assertThat(
+            recipientAddress.getAddress(),
+            Matchers.is(friend.email())
+        );
+    }
```
Comment (caarlos0) [inline netbout-web/src/test/java/com/netbout/email/EmAliasTest.java]: @mbarbieri you can do one assertion, comparing the arguments list directly to a list of InternetAddresses containing only friend.email()...

**Comment by caarlos0** [comment]
@mbarbieri couple of comments above

File: netbout-web/src/test/java/com/netbout/email/EmAliasTest.java (hunk 1 above)
Comment (mbarbieri) [inline netbout-web/src/test/java/com/netbout/email/EmAliasTest.java]: @caarlos0 you're right

**Comment by mbarbieri** [comment]
@caarlos0 done.
Thanks for the feedback

**Comment by caarlos0** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor  merge please

**Comment by dmzaytsev** [comment]
@rultor stop

**Comment by dmzaytsev** [comment]
@yegor256 If I have a right to some command to Rultor  why I can not cancel it?

**Comment by karato** [comment]
@ypshenychka please, let us know what do you think about this ticket, according to our [QA rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy pls

**Comment by ypshenychka** [comment]
@caarlos0 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. We should get used to this rule as soon as possible.
Please confirm that you will try to find at least three problems next time.

**Comment by caarlos0** [comment]
@ypshenychka sure, i'll try

**Comment by ypshenychka** [comment]
@caarlos0 Thank you!

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks for the QA review, we'll work better next time

**Comment by karato** [comment]
@caarlos0 10 mins was added to the account of @ypshenychka (for QA review), in transaction 58145841; Thanks a lot, I just topped your account for **19 mins**, transaction ID `AP-6DJ88053K2267621N`, total time was 16 hours and 6 mins; review comments (c=4) added as a bonus; added +19 to your rating, now it is equal to [+2165](http://www.netbout.com/b/35095?open=rating)

**Pull request merged**